app/src/org/commcare/dalvik/activities/EntitySelectActivity.java

```diff
@@ -788,6 +791,19 @@ private void select() {
         setResult(RESULT_OK, i);
         finish();
     }
+
+    // CommCare-159503: implementing DetailCalloutListener so it will not crash the app when requesting call/sms

```

Comment:
@orangejenny we do this a bunch on the HQ team, fyi. Agree is a bit odd, but definitely can be useful.